hieradata/vagrant/common.yaml

```diff
@@ -249,7 +239,7 @@ profile::jenkinscontroller::jcasc:
           virtualNetworkName: "public-vnet"
           virtualNetworkResourceGroupName: "public"
           subnetName: "public-vnet-ci_jenkins_io_agents"
-          javaHome: /opt/jdk-19       # Test override of the default JDK for builds
+          javaHome: /opt/jdk-11       # Test override of the default JDK for builds
```

Comment:
Do these tests still needed? Or are they stil "test"?

Reply:
yes, the comment is pretty self-explanatory. The goal is not to test the JDK11 or 19, but the goal is to validate the override mechanism does what it is expected to do.

Reply:
I understood what troubled me, I missed we're in tests here, not the prod.